acceptance/tests/mesh_v2/mesh_inject_test.go

```diff
@@ -50,6 +52,26 @@ func TestMeshInject_MultiportService(t *testing.T) {
 
 			consulCluster.Create(t)
 
+			consulClient, _ := consulCluster.SetupConsulClient(t, secure)
+
+			// Check that the ACL token is deleted.
+			if secure {
+				// We need to register the cleanup function before we create the deployments
+				// because golang will execute them in reverse order i.e. the last registered
+				// cleanup function will be executed first.
+				t.Cleanup(func() {
+					retrier := &retry.Timer{Timeout: 5 * time.Minute, Wait: 1 * time.Second}
+					retry.RunWith(retrier, t, func(r *retry.R) {
+						tokens, _, err := consulClient.ACL().TokenList(nil)
+						require.NoError(r, err)
+						for _, token := range tokens {
+							require.NotContains(r, token.Description, multiport)
+							require.NotContains(r, token.Description, connhelper.StaticClientName)
```

Comment:
~ Is there any constant or ID we can borrow directly from the test to make sure this check doesn't drift from token contents -> lead to a false negative?

Reply:
It's a great question - unfortunately, the only thing that contains the login metadata is the description. There are also other tokens in the system so we can't check for an empty set or a specific number.